src/main/scala/loamstream/drm/DrmaaPoller.scala

```diff
@@ -33,16 +33,16 @@ final class DrmaaPoller(client: DrmaaClient) extends Poller with Loggable {
     }
     
     //NB: Sort job ids for better log output
-    val sortedJobIds = jobIds.toSeq.sorted
+    lazy val sortedJobIds = jobIds.toSeq.sorted
     
-    debug(s"Polling status of jobs ${sortedJobIds.mkString(",")}")
+    trace(s"Polling status of jobs ${sortedJobIds.mkString(",")}")
     
-    val pollResults = sortedJobIds.map { jobId =>
+    val pollResults = Map.empty ++ sortedJobIds.iterator.map { jobId =>
```

Comment:
Just curious why these need to be sorted before checking status?

Reply:
It seems like the kind of thing that - if important - should be commented on so someone doesn't bork it in the future.

Reply:
I don't remember, but probably to facilitate easier-to-read logging.  It likely helps some tests too.